client/app/hosting/database/DATABASE_LIST.html

```diff
@@ -198,87 +198,41 @@
-                                            </button>
-                                        </li>
-                                        <li role="menuitem"
-                                            data-ng-if="!hosting.isCloudWeb && element.type === 'MYSQL'">
-                                            <button class="btn btn-link" type="button"
-                                                    data-i18n-static="hosting_tab_DATABASES_table_popover_check_quota"
-                                                    data-ng-click="ctrlTabDatabases.checkQuota(element)">
-                                            </button>
-                                        </li>
-                                        <li role="menuitem"
-                                            data-ng-if="element.type === 'MYSQL' || element.type === 'MONGODB'">
-                                            <button class="btn btn-link" type="button"
-                                                    data-i18n-static="hosting_tab_DATABASES_table_popover_delete"
-                                                    data-ng-click="setAction('database/delete/hosting-database-delete', element.name)">
-                                            </button>
-                                        </li>
-                                        <li role="menuitem"
-                                            data-ng-if="element.type === 'POSTGRESQL'">
-                                            <a class="btn btn-link d-inline-block text-left" target="_blank" title="{{ i18n.hosting_tab_DATABASES_table_popover_new_window }}"
-                                               data-i18n-static="hosting_tab_DATABASES_table_popover_phppgadmin"
-                                               data-ng-href="http://phppgadmin.ovh.net/">
-                                            </a>
-                                        </li>
-                                        <li role="menuitem"
-                                            data-ng-if="element.type === 'MYSQL'">
-                                            <a class="btn btn-link d-inline-block text-left" target="_blank" title="{{ i18n.hosting_tab_DATABASES_table_popover_new_window }}"
-                                               data-i18n-static="hosting_tab_DATABASES_table_popover_phpmyadmin"
-                                               data-ng-href="{{ ctrlTabDatabases.getPhpMyAdminUrl(element) }}">
-                                            </a>
-                                        </li>
-                                    </ul>
-                                </div>
+                                <oui-action-menu data-compact data-align="end">
+                                        <oui-action-menu-item data-text="{{:: tr('hosting_tab_DATABASES_table_popover_update_password') }}"
```

Comment:
You can nest a `span` instead of using `text` attribute as it is now deprecated

Reply:
Fixed on the last commit, thx for your comment 😃